WordPressAuthenticator/Unified Auth/View Related/Sign up/SignupMagicLinkViewController.swift

```diff
@@ -0,0 +1,197 @@
+import UIKit
+
+
+/// SignupMagicLinkViewController: step two in the signup flow.
+/// This VC prompts the user to open their email app to look for the magic link we sent.
+///
+final class SignupMagicLinkViewController: LoginViewController {
+
+    // MARK: Properties
+
+    @IBOutlet private weak var tableView: UITableView!
+    private var rows = [Row]()
+    private var errorMessage: String?
+    private var shouldChangeVoiceOverFocus: Bool = false
+
+    var emailMagicLinkSource: EmailMagicLinkSource?
```

Comment:
I don't see that `emailMagicLinkSource` is really used for anything. Will it be needed in the future, or is just carry-over fluff from the old VC?

Reply:
Accidental carry-over fluff. It was used to differentiate the "Magic Link sent" view as either `.signup` or `.login` for tracking purposes and whether or not to show the 1Password button. Will delete!